plugins/services/src/js/components/ServiceStatusProgressBar.js

```diff
@@ -0,0 +1,88 @@
+import React, { PropTypes } from "react";
+import { Tooltip } from "reactjs-components";
+
+import StringUtil from "#SRC/js/utils/StringUtil";
+import ProgressBar from "#SRC/js/components/ProgressBar";
+import Pod from "../structs/Pod";
+import Service from "../structs/Service";
+import ServiceTree from "../structs/ServiceTree";
+import ServiceStatus from "../constants/ServiceStatus";
+
+class ServiceStatusProgressBar extends React.Component {
+  getTooltipContent() {
+    const { service } = this.props;
+    const { tasksRunning } = service.getTasksSummary();
+    const instancesTotal = service.getInstancesCount();
+
+    return (
+      <div className="tooltip-line-item">
+        <span>
+          {`${tasksRunning} ${StringUtil.pluralize("instance", tasksRunning)} running out of ${instancesTotal} `}
+        </span>
+      </div>
+    );
+  }
+
+  getProgressBar(data) {
+    const { service } = this.props;
+    const instancesCount = service.getInstancesCount();
+    const tasksRunning = service.getTaskCount();
+
+    if (tasksRunning === instancesCount) {
+      return (
+        <ProgressBar
+          className="status-bar--large"
+          data={data}
+          total={instancesCount}
+        />
+      );
+    }
+
+    return (
+      <Tooltip interactive={true} content={this.getTooltipContent()}>
+        <ProgressBar
+          className="status-bar--large"
+          data={data}
+          total={instancesCount}
+        />
+      </Tooltip>
+    );
+  }
+
+  render() {
+    const { service } = this.props;
+    const instancesCount = service.getInstancesCount();
+    const serviceStatus = service.getStatus();
+    const tasksRunning = service.getTaskCount();
+
+    if (instancesCount !== tasksRunning) {
+      return this.getProgressBar([
+        {
+          state: "success",
+          value: tasksRunning
+        }
+      ]);
+    }
+
+    if (serviceStatus === ServiceStatus.DELETING.displayName) {
```

Comment:
Please adjust to align with comments on `StatusIcon` so we compare:
```
const status = service.getServiceStatus();
if (status === ServiceStatus.DELETING) {

}
```

Reply:
In this case `ServiceStatus.DELETING` is 
```
DELETING: {
    key: ServiceStatusTypes.DELETING,
    displayName: ServiceStatusLabels.DELETING
  }
```
which isn't what I want to compare

Did you mean `ServiceStatusLabels.DELETING`?

Reply:
It's OK to compare objects like this `status === ServiceStatus.DELETING` without digging down to the leaves of such an object tree to compare a string.